frontend/src/metabase/embedding-sdk/theme/dynamic-css-vars.ts

```diff
@@ -49,23 +75,21 @@ export function getDynamicCssVariables(theme: MantineTheme) {
       }
 
       // Do not define the CSS variable if the source color or operation is not defined.
-      if (!operation) {
+      // In addition, do not use chart color as source color to sample from.
+      if (!operation || operation?.source === "charts") {
         return [cssVar, null];
       }
 
-      let mappedColor = theme.fn.themeColor(operation.source);
+      // One SDK color will be mapped to multiple main app colors.
```

Comment:
This was where we previously generate some dynamic colors in the main app like `--mb-color-bg-sdk-question-toolbar`, mostly for SDK purposes. As I updated it to use the SDK color instead, we need to map here.